konflux-ci/rbac/core/konflux-admin-user-actions-core.yaml

```diff
@@ -33,34 +32,15 @@ rules:
     apiGroups:
       - appstudio.redhat.com
     resources:
-      - promotionruns
-      - snapshotenvironmentbindings
       - snapshots
-      - environments
-  - verbs:
-      - get
-      - list
-      - watch
-      - create
-      - update
-      - patch
-      - delete
-    apiGroups:
-      - appstudio.redhat.com
-    resources:
-      - deploymenttargets
-      - deploymenttargetclaims
   - verbs:
       - get
       - list
       - watch
     apiGroups:
-      - managed-gitops.redhat.com
+      - tekton.dev
     resources:
-      - gitopsdeployments
-      - gitopsdeploymentmanagedenvironments
-      - gitopsdeploymentrepositorycredentials
-      - gitopsdeploymentsyncruns
+      - taskruns
```

Comment:
this change is granting the right to create taskruns, it used to be get/list only before. Is this intentional?

Reply:
oh, sorry about that, I checked the permission block from after, got lost in the indentation...

Reply:
I would not assume anyone is directly using this permission there. @ralphbean, is that correct?